Dockerfile

```diff
@@ -3,10 +3,9 @@ ARG ALPINE_BASE_IMAGE="alpine:3.15"
 ARG BASE_REQUIREMENTS_FILE="requirements.txt"
 
 FROM ${NODE_BASE_IMAGE} as node_build
-COPY ./cla_public/static-src ./cla_public/static-src
 COPY ./package.json ./package-lock.json ./
 RUN npm install
-
+COPY ./cla_public/static-src ./cla_public/static-src
```

Comment:
This change is to prevent `RUN npm install` running anytime there is a change to files inside `./cla_public/static-src ` as changes to those files will not require an npm install. Instead a cache of the previous build will be used.

I think any time you do a `COPY ... ` changes to any files referenced in the COPY will result in all following Dockerfile steps to be rerun.

Reply:
Ahh brilliant, that makes sense. Thank you for the explanation!